src/cmd/tools.go

```diff
@@ -32,7 +32,7 @@ var toolsCmd = &cobra.Command{
 	Short: lang.CmdToolsShort,
 }
 
-// destroyCmd represents the init command
+// destroyCmd represents the init command.
 var archiverCmd = &cobra.Command{
 	Use:     "archiver",
```

Comment:
Isnt this the archiver command?

Reply:
Removed this comment since it didn't really make sense anyway and isn't needed.